dhis-2/dhis-web-api/src/main/resources/openapi/EnrollmentsExportController.md

```diff
@@ -6,58 +6,66 @@ Get enrollments matching given query parameters.
 
 ### `getEnrollments.parameter.enrolledAfter`
 
-Get enrollments enrolled after given date.
+Get enrollments with an enrollment date after the given date.
 
 ### `getEnrollments.parameter.enrolledBefore`
 
-Get enrollments enrolled before given date.
+Get enrollments with an enrollment date before the given date.
 
 ### `getEnrollments.parameter.enrollment`
 
 `<enrollment1-uid>[;<enrollment2-uid>...]`
 
-Get enrollments with given UID(s).
+Get enrollments with the given UID(s).
 
 ### `getEnrollments.parameter.followUp`
 
-Get enrollments with given follow-up status of the instance for the given program.
+Get enrollments with the given follow-up status.
 
 ### `getEnrollments.parameter.includeDeleted`
 
 Get soft-deleted enrollments by specifying `includeDeleted=true`. Soft-deleted enrollments are excluded by default.
 
 ### `getEnrollments.parameter.orgUnit`
 
-Get enrollments owned by given `orgUnit`.
+Get enrollments owned by given `orgUnit` relative to the `ouMode`. 
```

Comment:
The details of the relation between the ouMode and the org unit is only explained here and not in the ouMode description. It can perhaps be moved, but then the two parameters end up being described together in their entirety. Ca you have a look again and form an opinion?